modules/recovery/src/utils/deployment.ts

```diff
@@ -0,0 +1,9 @@
+export function getDeploymentParameters() {
```

Comment:
nit: since this is very tightly related to the deployment I'd keep it in `src/deploy` folder

Reply:
I just copied that pattern from the `safe-token`, but you are right that it isn't needed here (in the `safe-token`, these parameters were used for various tests as well, so it made sense to export them too).

Will move to the deployment script.

Reply:
Changed.